Cargo.toml

```diff
@@ -7,3 +7,12 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+ethcontract = "0.7.2"
+ethabi = "12.0.0"
```

Comment:
To not run into issues with crate version mismatches, I think it would be better to use `ethcontract::common::abi` which is a re-export of `ethabi`.